_data-prepper/monitoring.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Monitoring
+nav_order: 2
+---
+
+Metrics in Data Prepper are instrumented using [Micrometer.io](https://micrometer.io/). There are two types of metrics: (1) JVM and system metrics; (2) plugin metrics. Prometheus is used as the default metrics backend.
+
+# JVM and system metrics
+
+JVM and system metrics are Data Prepper instance runtime metrics including metrics on classloaders, memory, garbage collection, threads, etc. See https://micrometer.io/docs/ref/jvm for details.
+
+## Naming
+
+JVM and system metrics in Data Prepper follows pre-defined names in Micrometer.io, e.g. `jvm_classes_loaded`, `jvm_memory_used`.
```

Comment:
"... metrics ... follow ..."?
The Micrometer website appears to refer to itself as "Micrometer" alone (not with the ".io").
Spell out "for example" rather than using "e.g.": see "e.g." in TERMS.md.
Does this naming convention relate to the "Naming Meters" guidance in Micrometer documentation?
https://micrometer.io/docs/concepts#_naming_meters
Or is this a different convention? If this is an OpenSearch convention, we might want to explicitly describe the convention as using underscores to delineate terms. But if the convention at the Micrometer website I include here is accurate, you can link to that for explanation. (check with the SME?)